index.html

```diff
@@ -324,11 +351,35 @@
     .half-margin-bottom {
         margin-bottom: 50px;
     }
-    .name-text-container {
-        /*width: 700px;*/
-        /*padding: 0 100px;*/
-    }
 
+    .padding-container > img {
+        height: 100%;
+        width: 100%;
+    }
+    .padding-container {
+        padding: 25px;
+    }
+    .tech-container {
```

Comment:
Maybe on large screens we can drop the tech stack down to one row? Kinda nit picky, so do what you think is best.
![image](https://user-images.githubusercontent.com/16579931/96514134-11ad0980-1218-11eb-9db0-0e6992426da4.png)

Reply:
I think this is alright, only because most of the site looks awkward on very large screens. We could perhaps open an issue for setting up media queries for wide screens across the app.